flexmeasures/ui/__init__.py

```diff
@@ -133,6 +134,7 @@ def add_jinja_filters(app):
     )
     app.jinja_env.filters["asset_icon"] = asset_icon_name
     app.jinja_env.filters["username"] = username
+    app.jinja_env.filters["parsed_view_config_item"] = parse_applicable_viewname
```

Comment:
Why do you give the filter a different name than the function it calls?

Reply:
Good thought, I should use the same.